### ISSUE TITLE
r session interrupted when running library(rgl) on mac 

### ISSUE THREAD

**Issue by Agathedumontblanche** [dmurdoch/rgl#105]
![image](https://user-images.githubusercontent.com/84093653/118014656-bb3e1e80-b353-11eb-9db4-aa1dad3183d5.png)
* **rgl Version**: 0.106.8
* **R Version**: 1.4.1106
* **Platform**:   Mac OSX version: 10.15.7 (19H524) + XQuartz verssion: 2.8.1 (xorg-server 1.20.11) 

* To get around this we have removed the openGL in rgl

**Comment by dmurdoch** [comment]
That's not an R version number, it looks like an RStudio version number.  I'm using a slightly earlier one (1.4.1103).  

I'm using the same macOS version as you're using, but I'm still on XQuartz 2.8.0.

This year's updates to XQuartz have caused several problems with rgl:  you may need to compile rgl with the same version you have installed, and I'd guess the CRAN version wasn't using 2.8.1 when built:  rgl was last updated before 2.8.1 was released.

To help debug this, could you please post your R version number, and details about how you installed R and rgl?

**Comment by Agathedumontblanche** [comment]
In sumamry 

-Platform: Mac OSX version: 10.15.7 (19H524) + XQuartz verssion: 2.8.1 (xorg-server 1.20.11)

- I installed R from this link https://cran.r-project.org/bin/macosx/ in the application folder and the version is R 4.0.5 GUI 1.74 Catalina build (7950) , the version up to date 

- I installed rgl by running : wget https://cran.rproject.org/src/contrib/rgl_0.106.8.tar.gz
R CMD INSTALL --configure-args="--disable-opengl" rgl_0.106.8.tar.gz and the version is 0.106.8

thanks ;)

**Comment by dmurdoch** [comment]
I've updated everything now, and I'm not seeing the crash.  It does look like issues #27 and #43.  I'd suggest two things:

 - use `remotes::install_github("dmurdoch/rgl")` to install the latest development version.  It changes the startup process slightly in RStudio.

 - Try removing and reinstalling XQuartz according to these instructions:  https://github.com/dmurdoch/rgl/issues/43#issuecomment-794414896 .

Alternatively, if browser display is sufficient, just continue to work with --disable-opengl.

**Comment by gdagstn** [comment]
Hi, I am having basically the same problem but only with **`rgl` devel (0.108.27)**. Rstudio bombs and R segfaults in the terminal.

The current CRAN version of `rgl`  (0.108.3) works just fine in both cases. 

My specs are:
**R version**: 4.1.2
**XQuartz version**: 2.8.1 (xorg-server 1.20.11)
**Platform**: x86_64-apple-darwin17.0 (64-bit) - macOS Big Sur 11.6.4
**rgl version**: 0.108.27

I also tried doing what was suggested in #43 (uninstalling XQuartz and removing contents from /opt/X11), and it seemed to work; however, when I restarted the session, it went back to the usual behaviour.

I would really like to work with this version because it allows me to use `rgl2gltf` which needs `rgl` > 0.108.21. 

I suppose I don't need to have OpenGL functions enabled to create a `scene3d` that I can save to GLB, but having XQuartz capabilities is quite nice for browsing immediate results.
